tuftool/src/root.rs

```diff
@@ -106,6 +107,9 @@ pub(crate) enum Command {
         ///Optional - Path of older root.json that contains the key-id
         #[structopt(short = "c", long = "cross-sign")]
         cross_sign: Option<PathBuf>,
+        ///Force signing with fewer keys than the specified threshold
+        #[structopt(short = "f")]
```

Comment:
...just making sure: this is optional and defaults to `false`, right?

Reply:
That _seems_ to be the case from [structopt docs](https://docs.rs/structopt/0.3.23/structopt/#type-magic)... "`true` if the flag is present"...